client/packages/executor/src/gateways/evm/proof.ts

```diff
@@ -0,0 +1,120 @@
+import { BaseTrie as Trie } from "merkle-patricia-tree";
+import { encode, receiptToRlp } from "./encoder";
+import Web3, { Numbers, TransactionReceipt } from "web3";
+
+/**
+ * Construct the proof for a transaction, given its hash
+ *
+ * @param txHash the identifier of the transaction
+ * @return object { proof, tx index }
+ * */
+export const getProof = async (
+  txHash: string,
+  // client: Web3
+) => {
+  // Infura's endpoint. If it fails, generate a new one
+  const client = new Web3("https://sepolia.infura.io/v3/278f7a23305a406089f7dbfb7f4ef50d");
```

Comment:
Please move to ENV as this repo is now public. I have invalidated the token

Reply:
Moved.